gcp_variant_transforms/transforms/sample_info_to_bigquery.py

```diff
@@ -17,21 +17,30 @@
 import apache_beam as beam
 
 from gcp_variant_transforms.beam_io import vcf_header_io  # pylint: disable=unused-import
+from gcp_variant_transforms.beam_io import vcf_parser
 from gcp_variant_transforms.libs import sample_info_table_schema_generator
 from gcp_variant_transforms.libs import hashing_util
 
+SampleNameEncoding = vcf_parser.SampleNameEncoding
+
 
 class ConvertSampleInfoToRow(beam.DoFn):
   """Extracts sample info from `VcfHeader` and converts it to a BigQuery row."""
 
-  def process(self, vcf_header, samples_span_multiple_files):
+  def __init__(self, sample_name_encoding=SampleNameEncoding.WITHOUT_FILE_PATH):
```

Comment:
Let's remove the default value for `sample_name_encoding` here. I just want to make sure caller explicitly set this important flag.

Reply:
SG. Done.